src/main/webapp/WEB-INF/views/drivers/add.jsp

```diff
@@ -9,6 +9,11 @@
 </head>
 <body>
 <form method="post" id="driver" action="${pageContext.request.contextPath}/drivers/add"></form>
+<table class="table_dark">
+    <tr>
+        <th><%@include file="/WEB-INF/views/main/header.jsp" %></th>
+    </tr>
+</table>
 <h1 class="table_dark">Add driver:</h1>
 <table border="1" class="table_dark">
```

Comment:
are you use this jsp?

Reply:
Yes , I do. I use it when I press button "Add new driver" in main page, and then I go to register form , and there I can use this header "Back to main" if I don't want to add any new user .